lara-typescript/src/interactive-api-client/types.ts

```diff
@@ -297,6 +299,11 @@ export interface ICloseModal {
   uuid?: string;
 }
 
+export interface IDecoratedContentEventMessage {
+  type: string;
+  text: string;
+}
```

Comment:
As per my earlier suggestion, this could include:
```typescript
  bounds?: DOMRect;
```

Reply:
I like it.  Added.